### PR TITLE
Update probot to version > 7.1.1

### PR DESCRIPTION
**Pull request by jakewilkins**
GitHub is removing the [deprecated access token endpoint](https://developer.github.com/changes/2020-04-15-replacing-create-installation-access-token-endpoint/) which version 6 of Probot uses. There's a brownout scheduled this weekend which will impact reminders.

This updates Probot to >7.1.1. which is the earliest version to use the supported endpoint.

I'm new to the Probot ecosystem, so other than this `package.json` change I'm not sure what steps need to happen for a change like this. Since this is a major version change I imagine there's some other work that needs to be done but out of extreme optimism I'm opening this PR up to see how far it can take us.

🙏 ✨

### CODE REVIEW

**Comment by jbjonesjr** [comment]
@jakewilkins so i fixed the test framework to work, to show .... the tests no longer work.

createRobot can't be found. where is that method/function in the new probot?

**Comment by jakewilkins** [comment]
Thanks @jbjonesjr!

I'm not sure, I have ~0 familiarity with Probot. 7.1.1 isn't new Probot which is on version 9 now, 7.1.1 is the oldest version which supports the new endpoint. I was hoping to make the diff as small as possible to avoid some of these issues.

**Comment by jbjonesjr** [comment]
K. I'll review it further this weekend to see if I can find out what's up

**Comment by jakewilkins** [comment]
👋 Hey @jbjonesjr, were you able to look at this?

Also, heads up that the endpoint used by Reminders [will be removed on October 1st](https://developer.github.com/changes/2020-04-15-replacing-create-installation-access-token-endpoint/) 😬 contrary to [ismarchoveryet.com](https://ismarchoveryet.com) October starts next week.

The change is a 1 word change just pre-pending `apps/` to the path, if I were more familiar with Probot we might be able to skip the upgrade and just patch the version Reminders is using but I'm not sure if that's possible here.

File: package.json [hunk 1]
```diff
@@ -7,16 +7,21 @@
...
   },
   "dependencies": {
     "chrono-node": "^1.3.4",
     "moment": "^2.18.1",
     "parse-reminder": "^1.2.0",
-    "probot": "^6.0.0",
-    "probot-commands": "^1.1.0",
-    "probot-metadata": "^1.0.0",
-    "probot-scheduler": "^1.1.0"
+    "probot": "^10.9.0",
```
Comment (gr2m) [inline package.json:21]: oh wow, quite the jump 😱

**Review by gr2m — COMMENTED** [review]
The challenge with the app is that the tests are mocking `context.github` methods, instead of mocking http requests. Because of that tests continue to pass, even when methods such as `github.issues.edit` no longer exist. See also the first entry in the v10.0.0 release notes:
https://github.com/probot/probot/releases/tag/v10.0.0

Would you like me to help rewrite the tests to use `nock`? That will make it much easier to maintain the app moving forward

**Comment by jbjonesjr** [comment]
Bringing over notes from jakewilkins/reminders#1 

This PR now contains:
- Add support for vscode configs (and for easier codespaces usage!)
- Add support for GitHub actions
- Update probot and related libraries
- Fix unit tests for these changes
- Update unit test mocking for nock instead of jest
- various additional cleanups

**Pull request merged**